### PR TITLE
new zel library

### PR DESCRIPTION
**Pull request by cabutler10**
updates to react library
new typography page
updated libraries

NOTE: svg will not work until new svgs are pushed to cdn

### CODE REVIEW

**Comment by byjs-dev** [comment]
We have to adjust the ci: update node version. I'd like to create a complete ZEL package first and add it as well.

File: src/components/Header.jsx [hunk 1]
```diff
@@ -129,24 +162,86 @@ const Header = ({ handleMenu, zelTheme, handleTheme }) => {
...
-            placeholder="Select one"
-            label="Theme"
-            selectedItem={zelTheme}
-          />
+          <IconButton
+            aria-controls="simple-menu"
+            aria-haspopup="true"
+            onClick={handleClick}
+          >
+            <ZepIconEssentialSettings size="32" className={classes.icon} />
```
Comment (byjs-dev) [inline src/components/Header.jsx:197]: size should always be 24

File: src/components/Layout.jsx [hunk 2]
```diff
@@ -99,16 +98,16 @@ function Layout({ children }) {
...
   const h3Styled = props => (
-    <h3 className={clsx(classes.hStyled, 'zep-typo--normal-h3')} {...props} />
+    <h3 className={clsx(classes.hStyled, 'zep-typo--headline-3')} {...props} />
   )
   const pStyled = props => (
-    <p className={clsx(classes.pStyled, 'zep-typo--normal-body1')} {...props} />
+    <p className={clsx(classes.pStyled, 'zep-typo--body-default')} {...props} />
   )
   const hrStyled = () => (
     <hr className={clsx(classes.hrStyled, 'zep-border-color__gray-lighter')} />
```
Comment (byjs-dev) [inline src/components/Layout.jsx:113]: is 'zep-border-color__gray-lighter' set? AFAIK it's not part of zel.css anymore

File: src/components/code/CodeBlock.jsx [hunk 3]
```diff
@@ -3,15 +3,15 @@
 import React from 'react'
 import Highlight, { defaultProps } from 'prism-react-renderer'
 import CopyButton from '../button/CopyButton'
-import guidelines from 'zeppelin-element-library/guidelines.json'
+import guidelines from '@zlab-de/zel-react/guidelines.json'
```
Comment (byjs-dev) [inline src/components/code/CodeBlock.jsx:6]: Whats the difference between this guidelines.json files, shouldn't they be equal?

File: src/components/guidelines/IconSection.jsx [hunk 4]
```diff
@@ -113,35 +113,47 @@ function IconsSection() {
     setValue(newValue)
   }
 
-  const zelVersion = '0.11.4' // we cannot call from package.json as it is always on next version which is not published to cdn.
+  const zelVersion = '3.0.0' // we cannot call from package.json as it is always on next version which is not published to cdn.
```
Comment (byjs-dev) [inline src/components/guidelines/IconSection.jsx:116]: // we can ;)

File: src/components/radio/RadioExample.jsx [hunk 5]
```diff
@@ -16,14 +14,32 @@ function RadioExample() {
...
+          id="r1Inline"
+          name="value 1"
+          checked={radioInline === 'a'}
+          onChange={e => setRadioInline(e.target.value)}
+        />
+        <Radio
+          variant="inline"
+          value="b"
+          id="r2Inline"
+          name="value 2"
```
Comment (byjs-dev) [inline src/components/radio/RadioExample.jsx:40]: shouldn't radio buttons name values be the same inside a group?

File: src/content/zel-react/getting-started.mdx [hunk 6]
```diff
@@ -29,35 +29,117 @@ yarn add @zlab-de/zel-react
 
 ## How to use our components
 
-You must bring in **zel.css** and select a theme (theme-zeppelin, theme-cat, or theme-rental) to apply the styles from Zeppelin Element Library. It only needs to be brought it once at the root of your application.
+You must bring in **zel.css** and select a theme (theme-zeppelin, theme-cat, or theme-rental) to apply the styles from Zeppelin Element Library. It only needs to be brought it once at the root of your application. There is also a **guidelines.json** file to help you more easily access guidelines; colors, spacing, and typography specifications.
```
Comment (byjs-dev) [inline src/content/zel-react/getting-started.mdx:32]: Typo: It only needs to be brought it once -> ...brought in once

File: src/content/zel-react/getting-started.mdx [hunk 7]
```diff
@@ -29,35 +29,117 @@ yarn add @zlab-de/zel-react
...
+
+IT IS HIGHLY RECOMMENDED THAT THIS LIBRARY IS USED WITH ANOTHER COMPONENT LIBRARY LIKE MATERIAL UI.
```
Comment (byjs-dev) [inline src/content/zel-react/getting-started.mdx:34]: I'd put "To achieve comprehensive functionality ..." in front of this sentence.

File: src/data/elements.js [hunk 8]
```diff
@@ -5,25 +5,25 @@ module.exports = {
...
+          '<button class="zep-button zep-button--primary">Default</button>',
+          '<button class="zep-button zep-button--primary zep-button--medium">Medium</button>',
+          '<button class="zep-button zep-button--primary zep-button--small">Small</button>',
+          '<button class="zep-button zep-button--primary" disabled>Disabled</button>',
+          '<button class="zep-button zep-button--primary zep-button--icon"><svg width="32" height="32" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-labelledby="title_copy" class="zep-button__icon"><title id="title_copy">Copy</title><path fill="currentColor" d="M20,21 L9,21 L9,7 L20,7 L20,21 Z M20,5 L9,5 C7.8954305,5 7,5.8954305 7,7 L7,21 C7,22.1045695 7.8954305,23 9,23 L20,23 C21.1045695,23 22,22.1045695 22,21 L22,7 C22,5.8954305 21.1045695,5 20,5 L20,5 Z M17,1 L5,1 C3.8954305,1 3,1.8954305 3,3 L3,17 L5,17 L5,3 L17,3 L17,1 Z"></path></svg></button>',
```
Comment (byjs-dev) [inline src/data/elements.js:12]: SVG is an old one. Please use the latest from our icon lib. ...also on following buttons

**Review by byjs-dev — COMMENTED** [review]
I left some comments on a few files.
There are other  misconducts I didn't detect in code:
Typo does not transform on mobile breakpoint.
Typo "Display" is displayed as Roboto instead of RobotoCondensed.
Radio Element looks different than the radio button in ZEL and does not respond on label click.
Number Input example has no functionality and still the old icon svg.
Some code examples have still the old svg icon codes in it.
Pagination is different from ZEL Pagination ( old icon svg, wrong disabled state style)
Tag still has old close svg.

This is what I have noticed so far. Sorry for the long list. :(

**Comment by cabutler10** [comment]
@byjs-dev I fixed all code/frontend issues/tests, but the new circleci config file does not work. I attempted to fix as much as I can but now it complains about sonarcloud and aws so I can't figure out how to fix anymore. Whenever you get back could you could take a look or have someone else fix this.

**Comment by byjs-dev** [comment]
deployment issue will be fixed in the next step.

**Pull request merged**